cider-velox/test/CiderOperatorHashJoinTest.cpp

```diff
@@ -114,10 +114,11 @@ class CiderOperatorHashJoinTest : public CiderOperatorTestBase {
 TEST_F(CiderOperatorHashJoinTest, innerJoin_allTypes) {
   std::vector<TypePtr> types{BIGINT(), INTEGER()};
   for (const auto& type : types) {
-    testJoin({type},
```

Comment:
Hmm, a quick question: how to we ensure join is executed by Cider?
```
  void assertPlanConversion(const std::shared_ptr<const core::PlanNode>& plan,
                            const std::string& duckDbSql) {
    auto ciderPlan = CiderVeloxPluginCtx::transformVeloxPlan(plan);
    assertQuery(ciderPlan, duckDbSql);
  }
```
It seems only validating the result which may not be executed on Cider.

Reply:
CiderOp has been registered in presto-cpp logically and controlled by CiderPlanTransformer programatically.

Reply:
In this test, should we pass in some config to control it?

Reply:
We need enable join pattern in CiderPlanTransformer in the future. Current we disable it since there's still some other issues in integration test for join.